src/goose/batch.clj

```diff
@@ -1,22 +1,26 @@
 (ns goose.batch
   (:require
+    [goose.defaults :as d]
     [goose.utils :as u]))
 
 (def status-in-progress "in-progress")
```

Comment:
I missed this before, shall we change data structure of status from string to keyword?

Reply:
Changed them to keyword